samples/Dantooine/Dantooine.Client/Client/Properties/launchSettings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:54832",
+      "sslPort": 44382
```

Comment:
Please make sure this file doesn't reuse any assigned port 😃

Reply:
This is actual not used

Reply:
removed